server/src/main/java/org/apache/iotdb/db/mpp/plan/planner/distribution/ExchangeNodeAdder.java

```diff
@@ -447,9 +447,7 @@ private PlanNode processMultiChildNode(MultiChildProcessNode node, NodeGroupCont
         child -> {
           // If the child's region is NOT_ASSIGNED, it means the child do not belong to any
```

Comment:
It seems that this comment is not satisified the following logic any more, just remove it.